docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1034,8 +1044,8 @@ Table properties supply or set metadata for the underlying tables. This
 is key for :doc:`/sql/create-table-as` statements. Table properties are passed
 to the connector using a :doc:`WITH </sql/create-table-as>` clause::
 
-  CREATE TABLE tablename 
-  WITH (format='CSV', 
+  CREATE TABLE tablename
+  WITH (format='CSV',
```

Comment:
@mosabua please review doc changes.

@sravanip09 can you remove the redundant whitespaces in a separate PR?

Reply:
can you remove the redundant whitespaces in a separate PR?

Reply:
The problem is that many editors do not allow you to even save a file without trimming off the white space .. maybe just isolate that into a separate commit .. but I am even fine with leaving it as is